Bonsai.Tinkerforge/BrickletAirQuality.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.ComponentModel;
+using System.Reactive.Disposables;
+using System.Reactive.Linq;
+using Tinkerforge;
+
+namespace Bonsai.Tinkerforge
+{
+    [DefaultProperty(nameof(Uid))]
+    [Description("Measure IAQ (indoor air quality) index, air pressure, humidity and temperature from an Air Quality Bricklet.")]
+    public class BrickletAirQuality : Combinator<IPConnection, BrickletAirQuality.DataFrame>
+    {
+        [Description("The unique bricklet device UID.")]
+        [TypeConverter(typeof(UidConverter))]
```

Comment:
Can we move the `TypeConverter` line to above the description? This is entirely aesthetical, but since `Description` attributes tend to always be longer strings, they sit more comfortably close to the property name. It might also help associating the description with the property when reading.